pyproject.toml

```diff
@@ -32,15 +32,15 @@ https-loader = ["requests"]
 https-loader = "_delb.plugins.https_loader"
 
 [tool.poetry.dependencies]
-python = "~=3.6"
+python = ">=3.6.2,<4.0"
 cssselect = "*"
 lxml = "*"
 requests = {version = "^2.21", optional = true}
 setuptools = "*"
 
 [tool.poetry.dev-dependencies]
 autoclasstoc = "^1.1.1"
-black = {version = "*", allow-prereleases = true }
+black = "21.10b0"
```

Comment:
i'm puzzled how i updated black in the `poetry.lock` without touching this until now.

Reply:
me too, i tried several times.